### ISSUE TITLE
Playback restarts without user interaction after a while

### ISSUE THREAD

**Issue by pyby** [SRGSSR/playsrg-apple#159]
### Issue type

Incorrect behavior

### Description of the problem

Our testers reported a strange behaviour: The previous playback in paused mode, restarted without a user action.

> And one time it starts playing by magic, the mobile was on the table and starts playing. My phone was located at the table and starts playing I was 3m away. I will have an eye on it.

> Auto start playing... I had this strange behavior several times. I always thought it was my fault, by touching my AppleWatch which started the playing. If you do not have an AppleWatch, it becomes interesting...

I added similar issues few weeks ago. A livestream (RTS La 1ère) in pause mode, restarted I don't know how, in the pocket.

### Environment information

* Application version: 3.1.0 or more
* iOS version: users reported on iOS 13.x
* Device: iPhones

We should get more information, like Bluetooth connected devices, or other applications playing sounds or videos. It could be related to [Audio Session Interruptions](https://developer.apple.com/documentation/avfoundation/avaudiosession/responding_to_audio_session_interruptions?language=objc).

### Reproducibility

Difficult to reproduce

### Steps to reproduce

It would be nice to find a way to reproduce it.

**Comment by GrimbiXcode** [comment]
I can see the same behavior on iPhone Xr for video and audio playback. Restarting playback happens every time the iPhone is unlocked for the first time in the day near the Plantronics Bluetooth headset that is on standby on the cradle. The iPhone is not connected to the headset at this time. However, I suspect that information is then exchanged between the devices because the iPhone immediately uses the headset as an audio input and output device as soon as it is removed from the cradle.

I hope this helps to get to the source of the problem.

**Comment by defagos** [comment]
> I hope this helps to get to the source of the problem.

Thank you very much for your input. I'll have a look at BT headsets as well.

I also identified an issue with [network interruptions](https://github.com/SRGSSR/srgletterbox-apple/issues/231), which explains most complaints we receive since iOS 14 was released.

**Comment by defagos** [comment]
@GrimbiXcode I don't have the kind of headset model you have, so I tested with my own Bluetooth devices (AirPods and Bose QC II). The behavior I usually expect is correct: 

- Disconnecting the Bose headset or putting AirPods back into their case pauses playback. 
- Putting AirPods or connecting the Bose headset afterwards does not automatically resumes playback. 
- Unlocking or waking up the phone in presence of any of these devices does not resume playback either.

I experienced the same behavior with Apple Music app (which I use as reference in such cases).

I therefore conjecture that the unexpected restarts your encountered might be related to the problem [mentioned above](https://github.com/SRGSSR/srgletterbox-apple/issues/231). This problem leads to playback restarting in case of network drops, maybe this is what happens in your case as well at this very specific time in the day.

We'll deliver a fix for the unexpected restarts in the next version 3.1.4. I invite you to see whether you still experience this issue with this version.

Thanks again for your feedback, we'll keep an eye on it.
